### ISSUE TITLE
Run from a separate machine?

### ISSUE THREAD

**Issue by Noldaru** [skier233/nsfw_ai_model_server#13]
I have stash running (dockerized) on my home NAS (ubuntu) server, which evidently doesn't have a GPU. Would I be able to run the AI model from my desktop (manjaro) that does have one and connect that to the stash instance somehow?

**Comment by skier233** [comment]
Absolutely! This is a quite common setup that I've seen many people use. Basically you just need to ensure a few things:
1. The media files are accessible from your NAS server from your desktop
2. If the path to those files would be different from your desktop than the path would be that stash knows, you'll need to use the "path_mutation" option to "mutate" those paths into a format that will allow access from your desktop machine.

There's a bit of info in step 5 of the docker instructions here:
https://github.com/skier233/nsfw_ai_model_server/wiki/Installation-Instructions-(Docker)#building-the-docker-container

Basically if you're path on your NAS server to your media files are all something like C:/media/... and lets say that your NAS drive is mapped in on your desktop as F:/media/.. then in that path_mutation field you'd just want to do something like this:
```
path_mutation = {"C:/": "F:/"}
```

**Comment by Noldaru** [comment]
I apologize for the late reply, but thank you very much! I did end up getting it working thanks to that. 😃
